### PR TITLE
Feature/row item

### PR DESCRIPTION
**Pull request by ingkevinsoto**
Added RowItem
Added RowItem Story

### CODE REVIEW

File: components/table/RowItem.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (Cromeror) [inline components/table/RowItem.tsx:1]:
```suggestion
import * as React from "react";
```

File: components/table/rowItem.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.rowItem {
+  background-color: #FFF;
+
+  &:hover{
+    background-color: #BAE0FF;
+  }
+
+}
+.rowItemCell {
+  border-bottom: 1px solid #00000014;
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:10]: Please use the correct token for the border, the token is **_(paddingSmall)_**

File: components/table/rowItem.module.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.rowItem {
+  background-color: #FFF;
+
+  &:hover{
+    background-color: #BAE0FF;
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:5]: Use the correct token color.

File: components/table/rowItem.module.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.rowItem {
+  background-color: #FFF;
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:2]: use the correct token color.

File: components/table/rowItem.module.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+}
+.rowItemCell {
+  border-bottom: 1px solid #00000014;
+  display: inline-flex;
+  height: 48px;
+  min-height: 40px;
+  max-height: 68px;
+  padding: var(--padding-padding-small, 8px);
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:15]: Use the correct token for padding size

File: components/table/rowItem.module.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  align-items: center;
+  gap: var(--margin-margin-small, 8px);
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:17]: Use the correct token for margin size

File: components/table/rowItem.module.scss [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  flex-shrink: 0;
+}
+.selected {
+  background-color: #E6F4FF;
```
Comment (Cromeror) [inline components/table/rowItem.module.scss:21]: Use the correct token color.

File: components/table/rowItem.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  },
+};
+
+export const WithAntDIcons: Story = {
+  args: {
+    children: (
+      <Space>
+        <EditOutlined />
+        <DeleteOutlined />
+      </Space>
```
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:53]:
```suggestion
      <>
        <EditOutlined />
        <DeleteOutlined />
      </>
```

File: components/table/rowItem.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+export const WithAllAntDComponents: Story = {
+  args: {
+    children: (
+      <Space>
+        <Tag color="purple">Tag 1</Tag>
+        <EditOutlined />
+        <Input placeholder="Enter something" />
+      </Space>
```
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:71]:
```suggestion
      <>
        <Tag color="purple">Tag 1</Tag>
        <EditOutlined />
        <Input placeholder="Enter something" />
      </>
```

File: components/table/rowItem.stories.tsx [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+
+export const WithAntDTag: Story = {
+  args: {
+    children: (
+      <Space>
+        <Tag color="blue">Tag 1</Tag>
+        <Tag color="green">Tag 2</Tag>
+      </Space>
```
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:42]:
```suggestion
      <>
        <Tag color="blue">Tag 1</Tag>
        <Tag color="green">Tag 2</Tag>
      </>
```

File: components/table/rowItem.stories.tsx (hunk 8 above)
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:53]:
```suggestion
      <>
        <EditOutlined />
        <DeleteOutlined />
      </>
```

File: components/table/rowItem.stories.tsx [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+  title: "Atoms/RowItem",
+  component: RowItem,
+  tags: ["autodocs"],
+  argTypes: {
+    children: {
+      control: "text",
+      description: "Content to display inside the row item",
+    },
+  },
+} satisfies Meta<typeof RowItem>;
```
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:17]: Use `as` – that works better than `satisfies`.
```suggestion
} as Meta<typeof RowItem>;
```

File: components/table/rowItem.stories.tsx [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+export default meta;
+
+type Story = StoryObj<typeof meta>;
```
Comment (Cromeror) [inline components/table/rowItem.stories.tsx:20]: You need `import {StoryObj} from "@storybook/react";`

File: components/table/rowItem.module.scss (hunk 2 above)
Comment (ingkevinsoto) [reply components/table/rowItem.module.scss]: Done, but any way check the color

File: components/table/rowItem.module.scss (hunk 4 above)
Comment (ingkevinsoto) [reply components/table/rowItem.module.scss]: Done

File: components/table/rowItem.module.scss (hunk 5 above)
Comment (ingkevinsoto) [reply components/table/rowItem.module.scss]: Done

File: components/table/rowItem.module.scss (hunk 7 above)
Comment (ingkevinsoto) [reply components/table/rowItem.module.scss]: Done

File: components/table/rowItem.stories.tsx (hunk 9 above)
Comment (ingkevinsoto) [reply components/table/rowItem.stories.tsx]: Done

File: components/table/rowItem.stories.tsx (hunk 10 above)
Comment (ingkevinsoto) [reply components/table/rowItem.stories.tsx]: Done

File: components/table/rowItem.stories.tsx (hunk 11 above)
Comment (ingkevinsoto) [reply components/table/rowItem.stories.tsx]: Done

File: components/table/rowItem.stories.tsx (hunk 8 above)
Comment (ingkevinsoto) [reply components/table/rowItem.stories.tsx]: Done

File: components/table/rowItem.stories.tsx (hunk 12 above)
Comment (ingkevinsoto) [reply components/table/rowItem.stories.tsx:21]: Done

File: components/table/rowItem.module.scss (hunk 6 above)
Comment (ingkevinsoto) [reply components/table/rowItem.module.scss]: Done

**Review by Cromeror — APPROVED** [review]
Great Job!! 🚀

**Pull request merged**